python/openmldb/dbapi/dbapi.py

```diff
@@ -456,12 +454,15 @@ def get_resultset_schema(self):
 
 class Connection(object):
 
-    def __init__(self, db, zk, zkPath):
+    def __init__(self, db,is_cluster_mode=True, *args):
         self._connected = True
         self._db = db
-        self._zk = zk
-        self._zkPath = zkPath
-        options = sdk_module.OpenmldbSdkOptions(zk, zkPath)
+        self.zk_or_host = args[0]
```

Comment:
This is used for once. Remove `this` and do not set as object members.

Reply:
updated.